Platforms/QemuQ35Pkg/PlatformBuild.py

```diff
@@ -282,6 +282,7 @@ def SetPlatformEnv(self):
                           self.mws.join(self.ws, 'Platforms', 'QemuQ35Pkg', 'CfgData', 'Profile1QemuQ35PkgCfgData.csv'),
                           "Platform Hardcoded"
         )
+        self.env.SetValue('CONF_PROFILE_NAMES', "P0,P1", "Platform Hardcoded")

```

Comment:
Does this need set for QemuSbsaPkg also?

Reply:
Not for this one. Q35 had 2 profiles set, thus I want to pair the profile names with them. But SBSA does not seem to have any profiles, so I did not add any.